components/camera/videosource/logging/logger.go

```diff
@@ -36,8 +35,8 @@ func init() {
 	filePath = filepath.Join(config.ViamDotDir, "debug", "components", "camera", fmt.Sprintf("%s.txt", t))
 
 	var err error
-	if GLoggerCamComp, err = NewLogger(); err != nil && !errors.Is(err, UnsupportedError{}) {
-		log.Println("cannot create new logger: ", err)
+	if GLoggerCamComp, err = NewLogger(); err != nil {
+		logging.NewDebugLogger("camera").Info(err)
```

Comment:
[q] Do we intend to log `INFO` instead of `DEBUG` status here?

Reply:
Nope! Good catch. Changed.